lib/bozo/compilers/msbuild.rb

```diff
@@ -184,10 +188,36 @@ def framework_version
       framework_version
     end
 
+    def windows_path
+      to_unix_path(ENV['WINDIR'])
+    end
+
+    def default_msbuild_path(config)
+      File.join(windows_path, 'Microsoft.NET', config[:framework], config[:version], 'msbuild.exe')
+    end
+
+    def program_files
+      globalArch = ENV['PROCESSOR_ARCHITECTURE']
```

Comment:
Im not sure we want to take this directly from the ENV - build agents may be 64 bit and deployment may be 32 bit - at the least we should add an override